codeinsights-db/conf/postgresql.conf

```diff
@@ -0,0 +1,91 @@
+# -----------------------------
+# PostgreSQL configuration file
+# -----------------------------
+#
+# This file consists of lines of the form:
+#
+#   name = value
+#
+# (The "=" is optional.)  Whitespace may be used.  Comments are introduced with
+# "#" anywhere on a line.  The complete list of parameter names and allowed
+# values can be found in the PostgreSQL documentation.
+#
+# The commented-out settings shown in this file represent the default values.
```

Comment:
We can remove L13-L25 (maybe even L27-31 but eh).

Honestly I don't think we need to include any of the preamble - anyone who wants to configure Postgres will need to look up guidance anyway - but it's not a big deal.